src/components/ComponentList.js

```diff
@@ -134,7 +134,7 @@ export default class ComponentList extends React.Component {
   }
 
   renderRow({ index, key, style }, toggleExpanded = null, showExpanded = false) {
-    const { list, readOnly } = this.props
+    const { list, readOnly, githubToken } = this.props
```

Comment:
Connected components can just get it straight from the state, but it's a balancing act of what to keep independent (and thus possibly exported to NPM) what not to. Passing down stuff is a good way to get it down, and once we do have multiple tokens/whatnot we could pass a single tokens or config object down.

Reply:
I'd like to not have two pull requests touching the interface to `GitHubCommitPicker` open at the same time and I think it would be better to embed the token in `getGitHubRevisions` and have `SourcePicker` be a connected component. Could you look into https://github.com/clearlydefined/website/pull/313 first before we make decisions on this?